### PR TITLE
feat(form): add widget override for number separator

### PR DESCRIPTION
**Pull request by anehx**
image

Peek 2024-01-23 12-26

### CODE REVIEW

File: packages/form/addon/components/cf-field/input/number-separator.hbs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  type="text"
+  class="uk-input
+    {{if @field.isInvalid 'uk-form-danger'}}
+    {{if this.disabled 'uk-disabled'}}"
+  name={{@field.pk}}
+  id={{@field.pk}}
+  value={{this.displayValue}}
+  placeholder={{@field.question.raw.placeholder}}
+  readonly={{this.disabled}}
+  pattern={{this.pattern}}
```
Comment (derrabauke) [inline packages/form/addon/components/cf-field/input/number-separator.hbs:11]: There is no `this.pattern` in the component. Should it be an argument?

**Review by derrabauke — CHANGES_REQUESTED** [review]
One minor nitpick

File: packages/form/addon/components/cf-field/input/number-separator.hbs (hunk 1 above)
Comment (anehx) [reply packages/form/addon/components/cf-field/input/number-separator.hbs:11]: Leftover from something I tried. Removed it.

**Review by derrabauke — APPROVED** [review]
:+1:

**Pull request merged**